@xen-orchestra/web-core/lib/components/ui/radio-button/UiRadioButton.vue

```diff
@@ -0,0 +1,363 @@
+<!-- v4 -->
+<template>
+  <label class="ui-radio-button" v-bind="wrapperAttrs">
+    <input
+      v-model="radioModel"
+      :value="value"
+      :class="classNames"
```

Comment:
I would make the [same remark](https://github.com/vatesfr/xen-orchestra/pull/7993#discussion_r1830782133) as for the `TextArea` component, especially for the first point (i.e. where to add the variants’ classes). The CSS needs to be updated accordingly:
```css
.ui-radio-button {
/* Other styles */ 

  &.accent--info {
    .input {
      & + .fake-radio {
        border-color: var(--color-info-item-base);
      }

      &.indeterminate + .fake-radio {
        background-color: var(--color-info-item-base);

        &:hover {
          border-color: transparent;
          background-color: var(--color-info-item-hover);
        }

        &:active {
          background-color: var(--color-info-item-active);
        }
      }

      &:hover + .fake-radio {
        border-color: var(--color-info-item-hover);
      }

      &:focus-visible + .fake-radio::before {
        border: 0.2rem solid var(--color-info-txt-base);
      }

      &:active + .fake-radio {
        border-color: var(--color-info-item-active);
      }

      &:checked + .fake-radio {
        border-color: transparent;
        background-color: var(--color-info-item-base);
      }

      &:checked:hover + .fake-radio {
        background-color: var(--color-info-item-hover);
      }

      &:checked:active + .fake-radio {
        background-color: var(--color-info-item-active);
      }

      &:disabled {
        & + .fake-radio {
          cursor: not-allowed;
          border-color: var(--color-neutral-txt-secondary);

          > .icon {
            color: var(--color-neutral-txt-secondary);
          }
        }

        &:checked + .fake-radio {
          border-color: transparent;
          background-color: var(--color-info-item-disabled);
        }
      }
    }
  }

  &.accent--success {/*...*/}

 /* Other variants */
}
```